funcions/10.test.js

```diff
@@ -0,0 +1,11 @@
+const { Calculator } = require('./10')
+
+describe('calc', () => {
+	it('calling add(5).multiply(2).add(20).divide(3) on calc returns 10', () => {
```

Comment:
Consider prefixing spec names with `should ` just to enforce a consistent naming convention :)

Reply:
What are the spec names exactly?

Reply:
Zanas meant to suggest creating more readable test names, like:
```
describe('test Calculator', () => {
  it('should return 10 when "add(5).multiply(2).add(20).divide(3)" is called on "calc"')
```
It's just a nitpick, but `it should` reads really well 😃